events/views.py

```diff
@@ -127,6 +135,9 @@ def form_invalid(self, form):
 def ws_send(request, form, public_info):
     ws_schema = 'ws' if settings.DEVELOP else 'wss'
     url = request.META.get('HTTP_HOST')
+    # This indicates that the request is being ran by the tests
+    if not url:
```

Comment:
We could also put something like this: `TESTING = sys.argv[1:2] == ['test']` into core/settings.py and do `if TESTING:` wherever testing specific code is run.

Reply:
Reverted the incorrect unpublished event check and added the suggested TEST environment variable